aws/lambdas/file_upload_processor.tf

```diff
@@ -0,0 +1,59 @@
+#
+# Form File Upload API processing
+#
+
+
+resource "aws_lambda_function" "file_upload" {
+  function_name = "file-upload-processor"
+  image_uri     = "${var.ecr_repository_lambda_urls["file-upload-processor-lambda"]}:latest"
+  package_type  = "Image"
+  role          = aws_iam_role.lambda.arn
+  timeout       = 300
+  # Only allow a single instance to run at a time
+  reserved_concurrent_executions = 1
```

Comment:
We will have to see how it goes once we have to handle lots of submissions with attached files. Obviously it won't prevent people from submitting but will delay the availability of submissions to form owners.